libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -674,40 +691,63 @@ export const DealTicket = ({
+                name="iceberg"
+                control={control}
+                render={({ field }) => (
+                  <Tooltip
+                    description={
+                      <p>
+                        {t(
+                          'ICEBERG_TOOLTIP',
+                          'Trade only a fraction of the order size at once. After the peak size of the order has traded, the size is reset. This is repeated until the order is cancelled, expires, or its full volume trades away. For example, an iceberg order with a size of 1000 and a peak size of 100 will effectively be split into 10 orders with a size of 100 each. Note that the full volume of the order is not hidden and is still reflected in the order book.'
+                        )}{' '}
+                        <ExternalLink href={DocsLinks?.ICEBERG_ORDERS}>
+                          {t('Find out more')}
+                        </ExternalLink>{' '}
+                      </p>
+                    }
+                  >
+                    <div>
+                      <Checkbox
+                        name="iceberg"
+                        checked={field.value}
+                        onCheckedChange={field.onChange}
+                        disabled={disableIcebergCheckbox}
+                        label={t('Iceberg')}
+                      />
+                    </div>
+                  </Tooltip>
+                )}
+              />
+            )}
             <Controller
-              name="iceberg"
+              name="tpSl"
               control={control}
               render={({ field }) => (
                 <Tooltip
                   description={
-                    <p>
-                      {t(
-                        'ICEBERG_TOOLTIP',
-                        'Trade only a fraction of the order size at once. After the peak size of the order has traded, the size is reset. This is repeated until the order is cancelled, expires, or its full volume trades away. For example, an iceberg order with a size of 1000 and a peak size of 100 will effectively be split into 10 orders with a size of 100 each. Note that the full volume of the order is not hidden and is still reflected in the order book.'
-                      )}{' '}
-                      <ExternalLink href={DocsLinks?.ICEBERG_ORDERS}>
-                        {t('Find out more')}
-                      </ExternalLink>{' '}
-                    </p>
+                    <p>{t('TP_SL_TOOLTIP', 'Take profit / Stop loss')}</p>
                   }
                 >
                   <div>
                     <Checkbox
-                      name="iceberg"
+                      name="tpSl"
                       checked={field.value}
                       onCheckedChange={field.onChange}
-                      disabled={disableIcebergCheckbox}
-                      label={t('Iceberg')}
+                      disabled={false}
+                      label={t('TP / SL')}
```

Comment:
'TP / SL' not added to i18n/src/locales/en/deal-ticket.json

Reply:
updated: added translations